src/pages/Home/Home.js

```diff
@@ -8,4 +8,4 @@ const Home = () => {
   );
 };
 
-export default Home;
+export default Home;
```

Comment:
Please keep the new line in this file
![Uploading Screen Shot 2020-11-11 at 10.03.58 AM.png…]()